scripts/health_metrics/create_binary_size_report.sh

```diff
@@ -19,43 +19,49 @@
 # This will generate a JSON output of changed files and their newly added
 # lines.
 
```

Comment:
It's a bit difficult to understand how to use the script. Could you please either:
- replace environment variable with the script input parameters and provide docs on the usage explaining the parameters
- provide docs on the environment variables that are used as input parameters